src/components/views/rooms/RoomHeader.tsx

```diff
@@ -57,6 +86,41 @@ export default function RoomHeader({ room }: { room: Room }): JSX.Element {
                     </BodyText>
                 )}
             </div>
+
+            {!isDirectMessage && (
+                <BodyText
+                    as="div"
+                    size="sm"
+                    weight="medium"
+                    className="mx_RoomHeader_members"
+                    aria-label={_t("%(count)s members", { count: memberCount })}
+                    onClick={(e: React.MouseEvent) => {
+                        togglePanel(RightPanelPhases.RoomMemberList);
+                        e.stopPropagation();
+                    }}
+                >
+                    <AvatarStack>
+                        {members.slice(0, 3).map((member) => {
+                            const mxcUrl = member.getMxcAvatarUrl();
+                            return (
+                                <Avatar
+                                    key={member.userId}
+                                    id={member.userId}
+                                    name={member.name}
+                                    src={
+                                        mxcUrl
+                                            ? mediaFromMxc(mxcUrl, client).getThumbnailOfSourceHttp(32, 32, "crop") ??
+                                              undefined
+                                            : undefined
+                                    }
+                                    size="20px"
+                                />
```

Comment:
By mixing avatar components here (not using things derived from our BaseAvatar) won't we have different looking default avatars?

Reply:
Not using BaseAvatar here means it won't reload correctly if loaded during a network outage and it won't handle the `lowBandwidth` setting. See `useImageUrl` at the very least

Reply:
We will. The `Avatar` component has not been finalised on Compound's side and i'm waiting to have the full specs before committing to changing it in Element Web.

For now it's fine as this is in labs.

Reply:
Can we have a red flag `// XXX: switch BaseAvatar over to Compound Avatar before we ship` so it doesn't get forgotten please